financialaid/views.py

```diff
@@ -48,7 +48,7 @@ class ReviewFinancialAidView(UserPassesTestMixin, ListView):
     View for reviewing financial aid requests.
     Note: In the future, it may be worth factoring out the code for sorting into its own subclass of ListView
     """
-    paginate_by = 10
+    paginate_by = 2
```

Comment:
Ah, I changed it to 50, then to 2 to test sorting with pagination, then forgot to change it back.  Thanks!